serverless/packages/functions/src/datasets/user/delete_user_uploaded_data.ts

```diff
@@ -0,0 +1,29 @@
+import { APIGatewayProxyHandlerV2 } from "aws-lambda";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import { S3Client, DeleteObjectCommand } from "@aws-sdk/client-s3";
+import parseJwt from "@dlp-sst-app/core/parseJwt";
+
+export const handler: APIGatewayProxyHandlerV2 = async (event) => {

```

Comment:
rename files so that they are less verbose (e.g. `delete_user_uploaded_data` can just be `delete`)

Reply:
actually delete_url would probably be a better name for this

Reply:
agreed